internal/web_server/web/handlers/api/admin_api.go

```diff
@@ -127,6 +127,16 @@ func (aah *AdminApiHandler) editQuizImage(c echo.Context) error {
 		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Ugyldig bilde URL"))
 	}
 
+	imageName, err := aah.uploadImageFromURL(c, *imageURL)
```

Comment:
Is there any security against uploading the image if the URL is already to the bucket?

Reply:
as best as i can understand, the file will just be replaced, tho that is unlikely given the nature of uuidv4